motor/docstrings.py

```diff
@@ -1219,3 +1295,285 @@ async def coro():
         encrypted = await client_encryption.encrypt(value, ...)
         decrypted = await client_encryption.decrypt(encrypted)
 """
+
+gridfs_delete_doc = """Delete a file's metadata and data chunks from a GridFS bucket::
```

Comment:
Let's add the session parameter to these gridfs docstrings.

Reply:
Done